internal/security/proxy/service.go

```diff
@@ -132,7 +142,18 @@ func (s *Service) Init(cp bootstrapConfig.CertKeyPair) error {
 		}
 	}
 
-	for clientName, client := range s.configuration.Clients {
+	// map[string]bootstrapConfig.ClientInfo
+	addRoutesFromEnv, parseErr := s.parseAdditionalProxyRoutes()
+
+	if parseErr != nil {
+		s.loggingClient.Error(fmt.Sprintf(
+			"failed to parse additional proxy Kong routes from env %s: %s",
+			s.additionalRoutes, parseErr.Error()))
+	}
+
+	mergedClients := s.mergeMapWith(addRoutesFromEnv)
```

Comment:
`mergeMapWith` is too generic. How about `mergeRoutesWith`?

Reply:
ok